static/src/javascripts/projects/commercial/sentinel.ts

```diff
@@ -1,38 +1,63 @@
-import raven from 'lib/raven';
 import config_ from '../../lib/config';
 
 // This is really a hacky workaround ⚠️
 const config = config_ as {
 	get: (s: string, d: boolean) => boolean;
 };
 
+export type SentinelLoggingEvent = {
+	received_timestamp: Date;
+	received_date: string;
+	label: string;
+	properties?: Property[];
+};
+
+type Property = {
+	name: string;
+	value: string;
+};
+
 /**
- * This function is used to send a logging event to Sentry.
- * Among other things, Sentry tracks the URL of the page where this function
- * is executed, which will allow developers to easily identify pages where
- * obscure commercial code is run.
- * Please note: due to the Raven configuration in frontend, all events generated
- * by this function will be logged in project 35463 (client-side-prod) on Sentry.
- *
- * @param moduleName the name of the JS/TS module (ex: 'article-aside-adverts')
- * @param functionName the name of the function within the module (ex: 'init')
- * @param label an optional value to add function-specific information (ex: 'conditionA=true')
+ * This function is used to send a logging event to BigQuery, which is
+ * logged into the `fastly_logging` table within the `logging` dataset.
+ * @param moduleName the name of the JS/TS module (e.g. `article-body-adverts`)
+ * @param functionName the name of the function within the module (e.g. `init`)
+ * @param parameters an optional object to add function-specific information (e.g. `{ conditionA: true, conditionB: false }`)
  * @returns void.
  */
 export const amIUsed = (
 	moduleName: string,
 	functionName: string,
-	label?: string,
+	parameters?: Record<string, unknown>,
```

Comment:
This might lead to the the extra properties overriding the native ones.

[Maybe something like this could help?][ts]

```ts
type ExtraProperties = Record<string, string> & {
    module_name: undefined;
    function_name: undefined;
    URL: undefined;
}
```

[ts]: https://www.typescriptlang.org/play?#code/C4TwDgpgBAogHsATgQwAqIPaUcAlhAZygF4oAlCAYw0QBMAeApXAOwHMAaKJxVtgPigAyKAG8AUFClQAthloBXADYQA+i2QyIALigKWtCADNWEWgG5J0o-sp4MLdZp16Dx0xatSAqmQAyuvqGJixmlgC+4uLULExQAEa68EhomNh4hCRiXlDUMloswLoA5DIguRj5EIXFHDk2LHa4Dk5aJTx8tTm+AVDFRsgA1hAAdAqISsXi4eZAA